elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -186,9 +186,14 @@ export class TransactionPool {
 
 		// Remove from feePriorityQueue
 		this._feePriorityQueue.clear();
-		for (const node of this._feePriorityQueue.getAllNodes()) {
-			if (node.value !== foundTx.id) {
-				this._feePriorityQueue.push(node.key, node.value);
+		for (const txObject of this.getAllTransactions()) {
+			if (txObject.id !== foundTx.id) {
```

Comment:
since it's removed already above, it will never be `===`

Reply:
yes 😅